### PR TITLE
Specific date checkbox

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: report.20191210.155319.4601.0.001.json [hunk 1]
```diff
@@ -0,0 +1,807 @@
+
```
Comment (lbiedinger) [inline report.20191210.155319.4601.0.001.json]: this file shouldn't be in the PR

File: components/search/DateFilter.vue [hunk 2]
```diff
@@ -107,12 +124,24 @@
...
+      },
+      specific() {
+        this.form.specific = this.specific;
       }
     },
 
     methods: {
       emitDateForm() {
-        this.$emit('dateFilter', this.name, this.form);
+        Vue.nextTick(() => { // Change event triggers before v-model has updated. This resolves the issue
```
Comment (rwd) [inline components/search/DateFilter.vue]: The contents of this function can be reduced to simply:
```js
this.$emit('dateFilter', this.name, this.form);
```
... with the changes to the event handler in the parent component noted below.

File: components/search/MoreFiltersDropdown.vue [hunk 3]
```diff
@@ -148,8 +153,18 @@
       },
 
       dateFilterSelected(facetName, dateRange) {
```
Comment (rwd) [inline components/search/MoreFiltersDropdown.vue]: Change this function's logic to:
```js
        let dateQuery = [];

        if (dateRange.specific) {
          if (dateRange.start) {
            dateQuery = [dateRange.start];
          }
        } else if (dateRange.start || dateRange.end) {
          dateQuery = [rangeToQueryParam(dateRange)];
        }

        this.isCheckedSpecificDate = dateRange.specific;

        this.updateSelected(facetName, dateQuery);
```
It's a bit more concise and supports the above changes to emitDateForm.

File: components/search/FacetFieldLabel.vue [hunk 4]
```diff
@@ -53,6 +61,17 @@
         let subtype = this.fieldValue.split('/')[1];
 
         return subtype.replace(/^x-/, '').toUpperCase();
+      },
+
+      dateLabel() {
```
Comment (rwd) [inline components/search/FacetFieldLabel.vue]: This is redundant and can be removed.

**Pull request merged**